ledger/participant-integration-api/src/main/scala/platform/store/dao/events/ContractWitnessesTable.scala

```diff
@@ -3,64 +3,40 @@
 
 package com.daml.platform.store.dao.events
 
+import java.sql.Connection
+
 import anorm.{BatchSql, NamedParameter}
 import com.daml.platform.store.Conversions._
 import com.daml.platform.store.DbType
 
-private[events] sealed abstract class ContractWitnessesTable {
-
-  protected val insert: String
+private[events] abstract class ContractWitnessesTable {
 
   protected val TableName = "participant_contract_witnesses"
   protected val IdColumn = "contract_id"
   protected val WitnessColumn = "contract_witness"
 
-  private def toNamedParameter(idAndParty: (ContractId, Party)) =
-    idAndParty match {
-      case (id, party) =>
-        List[NamedParameter](
-          IdColumn -> id,
-          WitnessColumn -> party,
-        )
-    }
-
-  private def prepareBatchInsert(witnesses: WitnessRelation[ContractId]): Option[BatchSql] =
-    batch(insert, Relation.flatten(witnesses).map(toNamedParameter).toSeq)
-
   protected val delete = s"delete from $TableName where $IdColumn = {$IdColumn}"
```

Comment:
This field is not used in children. Please make it private

Reply:
fixed